qiskit/extensions/unitary.py

```diff
@@ -199,6 +180,52 @@ def qasm(self):
         return self._qasm_definition + self._qasmif(self._qasm_name)
 
 
+def _compute_control_matrix(base_mat, num_ctrl_qubits, ctrl_state=None):
+    """
```

Comment:
```suggestion
    r"""
```
Raw strings should resolve the invalid escape characters and enable latex comments